src/ds/json/adv-jsonpath.md

```diff
@@ -0,0 +1,191 @@
+[JSONPath](https://goessner.net/articles/JsonPath/) expressions help you access specific elements within a JSON document, which is similar to how XPATH works for XML documents.
+JSONPath support was added to Redis Stack in version 2.0.
+Before that, [a legacy form of pathing](https://redis.io/docs/data-types/json/path/#legacy-path-syntax) was supported.
+Only JSONPath will be discussed in this tutorial.
+
+You've already seen several examples of JSONPath in previous parts of this tutorial. The next sections will describe JSONPath in more complete detail.
+Some simple JSON documents will be used to demonstrate JSONPath's features.
+
+```redis Load documents
+JSON.SET lit1 $ 5
+JSON.SET lit2 $ '"abc"'
+JSON.SET lit3 $ true
+JSON.SET lit4 $ null
+JSON.SET obj1 $ '{"a":1, "b":2}'
+JSON.SET obj2 $ '{"a":[1,2,3,"a","b","c",false,true,["a",1],{"a":1},{"b":null}], "b":5}'
+JSON.SET arr1 $ [1,2,3]
+JSON.SET arrmap $ '[{"a":1}, {"b":2}]'
+```
+
+The returned value of a JSONPath expression is a JSON string with a top-level array of JSON serialized strings.
```

Comment:
What does it mean 'the returned value of a JSONPath expression'?  Do you mean the return value of the `JSON.GET` command? A JSONPath expression resolves into absolute path matches.

Reply:
I removed the sentence as it didn't really add any value.